src/brisk.jl

```diff
@@ -0,0 +1,106 @@
+"""
+```
+brisk_params = BRISK([pattern_scale = 1.0])
+```
+
+| Argument | Type | Description |
+|----------|------|-------------|
+| `pattern_scale` | `Float64` | Scaling factor for the sampling window | 
+"""
+type BRISK <: Params
+    threshold::Float64
+    octaves::Int
+    pattern_scale::Float64
+    pattern_table::Vector{Vector{SamplePair}}
+    smoothing_table::Vector{Vector{Float16}}
+    orientation_weights::Vector{OrientationWeights}
+    short_pairs::Vector{OrientationPair}
+    long_pairs::Vector{OrientationPair}
```

Comment:
Right, as long as you're using leaftypes, it doesn't matter whether you have parameters or not.